<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeedToTime3</class>
 <widget class="QWidget" name="SeedToTime3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gen 3 Seed to Time</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="seedLabel">
     <property name="text">
      <string>16/32-Bit Seed</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="TextBox" name="textBoxSeed"/>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelYear">
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="TextBox" name="textBoxYear">
     <property name="text">
      <string notr="true">2000</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelAdvances">
     <property name="text">
      <string>Advances</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="TextBox" name="textBoxAdvances">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButtonFind">
     <property name="text">
      <string>Find</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="TableView" name="tableView">
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Form/Controls/TextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Form/Controls/TableView.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
